packages/toolpad-utils/src/fs.ts

```diff
@@ -50,12 +50,16 @@ export async function writeFileRecursive(
   await fs.writeFile(filePath, content, options);
 }
 
-export async function updateYamlFile(filePath: string, content: object) {
+export async function updateYamlFile(
+  filePath: string,
+  content: object,
+  callback?: (content: string) => string,
```

Comment:
I'd keep the `updateYamlFile` interface as it was and call `prettier` inside of it. We'd have to add `prettier` as a dependency to `@mui/toolpad-utils`. Just make sure it's the same version as the rest of the project.

Reply:
My problem.
Actually, I wrote both ways and felt that could reduce dependency.
I will fix it late

Reply:
No worries. There shouldn't be any extra dependencies installed in the user's node_modules fodler if we keep the same version across the whole project.